packages/suite/src/support/messages.ts

```diff
@@ -8820,4 +8820,8 @@ export default defineMessages({
         id: 'TR_STAKE_CONSENT_TO_STAKING_WITH_EVERSTAKE',
         defaultMessage: 'I acknowledge and consent to staking with Everstake',
     },
+    TR_SOLANA_TX_CONFIRMATION_MAY_TAKE_UP_TO_1_MIN: {
+        id: 'TR_SOLANA_TX_CONFIRMATION_MAY_TAKE_UP_TO_1_MIN',
+        defaultMessage: 'Please wait, confirmation of the transaction may take up to 1\u00a0minute',
```

Comment:
`\u00a0` for `NO-BREAK SPACE` 

- [ ] Todo: figure out how to do this properly